includes/polling/ports.inc.php

```diff
@@ -445,15 +445,12 @@
             $port['update']['poll_period'] = $polled_period;
         }
 
-        // Copy ifHC[In|Out]Octets values to non-HC if they exist
-        if ($this_port['ifHCInOctets'] > 0 && is_numeric($this_port['ifHCInOctets']) && $this_port['ifHCOutOctets'] > 0 && is_numeric($this_port['ifHCOutOctets'])) {
-            echo 'HC ';
-            $this_port['ifInOctets']  = $this_port['ifHCInOctets'];
+        // use HC values if they are available
+        if (is_numeric($this_port['ifHCInOctets']) && $this_port['ifHCInOctets'] >= 0) {
```

Comment:
Isn't that still the same issue as @geordish just pointed out that we actually haven't polled for ifInOctets at that point in some situations because we've already decided that HC counter being 0 is valid (which apparently isn't always the case).